streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1171,72 +1170,6 @@ public class StreamsConfig extends AbstractConfig {
         CONSUMER_EOS_OVERRIDES = Collections.unmodifiableMap(tempConsumerDefaultOverrides);
     }
 
-    public static class InternalConfig {
```

Comment:
Even is this class it's public, it's called `InternalConfig` and thus clearly internal -- I believe it's ok to move to new sub-class without deprecation (as a matter of fact, we won't need this helper class any longer, and I only moved the members of this class to `InternalStreamsConfig`.